src/dotnet/APIView/APIViewWeb/Client/css/site.scss

```diff
@@ -757,6 +757,15 @@ code {
     text-decoration: line-through;
 }
 
+.hidden-api .code-inner {
+  color: var(--text-color-muted) !important;
+  font-style: italic;
+  span, a {
```

Comment:
I needed to add the more specific selectors to get the styles to take precedence over the various token styles.